vt100_output_windows.go

```diff
@@ -0,0 +1,299 @@
+// +build windows
```

Comment:
go-colorable handles most of escape sequence(s) on Windows.

https://github.com/mattn/go-colorable/blob/master/colorable_windows.go